File: ArkhamAsylum.Lib/Models/NurseRoomAssignation.cs
using Newtonsoft.Json;
using System;

namespace ArkhamAsylum.Lib.Models
{
    public class NurseRoomAssignation : Entity
    {
        public string Code { get; set; }
        public Guid NurseId { get; set; }
        [JsonIgnore]
        public Nurse Nurse { get; set; }
        public Guid RoomId { get; set; }
        [JsonIgnore]
        public Room Room { get; set; }
    }
}

File: ArkhamAsylum.Web/wwwroot/Scripts/Models/NurseRoomAssignation.js
class NurseRoomAssignation extends Entity
{
    constructor(json)
    {
        super(json)
        if (json)
        {
            this.Code = json.code;
            this.NurseId = json.nurseId;
            this.RoomId = json.roomId;
        }
        else
        {
            this.Code = "";
            this.NurseId = "00000000-0000-0000-0000-000000000000";
            this.RoomId = "00000000-0000-0000-0000-000000000000";
        }
    }
}